Makefile

```
@@ -17,21 +17,21 @@
 #   make install
 #
 
+VERSION ?= $(shell git describe --tags --always --dirty 2>/dev/null || echo "dev")
 DIR := $(dir $(abspath $(firstword $(MAKEFILE_LIST))))
-PGEDGE_ANSIBLE_VERSION ?= $(shell cat $(DIR)/VERSION | head -n 1)
-TARGET := $(DIR)/pgedge-platform-$(PGEDGE_ANSIBLE_VERSION).tar.gz
+TARGET := $(DIR)/pgedge-platform-$(VERSION).tar.gz
 
 .PHONY: build clean install
 
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)